charts/spire/charts/spiffe-oidc-discovery-provider/templates/configmap.yaml

```diff
@@ -1,44 +1,43 @@
 {{- $oidcSocket := "/run/spire/oidc-sockets/spire-oidc-server.sock" }}
+{{- define "spiffe-oidc-discovery-provider.yaml-config" -}}
+log_level: {{ .Values.config.logLevel | quote }}
+
+domains:
+  - "{{ include "spiffe-oidc-discovery-provider.fullname" . }}"
+  - "{{ include "spiffe-oidc-discovery-provider.fullname" . }}.{{ include "spiffe-oidc-discovery-provider.namespace" . }}"
+  - "{{ include "spiffe-oidc-discovery-provider.fullname" . }}.{{ include "spiffe-oidc-discovery-provider.namespace" . }}.svc.{{ include "spiffe-oidc-discovery-provider.cluster-domain" . }}"
+  {{- if gt (len .Values.config.domains) 0 }}
+  {{- .Values.config.domains | toYaml | nindent 2 }}
+  {{- end }}
+
+{{- if .Values.insecureScheme.enabled }}
+allow_insecure_scheme: {{ .Values.insecureScheme.enabled }}
+listen_socket_path: {{ .oidcSocket | quote }}
+{{- else }}
+acme:
+  directory_url: {{ .Values.config.acme.directoryUrl | quote }}
+  cache_dir: {{ .Values.config.acme.cacheDir | quote }}
+  tos_accepted: {{ .Values.config.acme.tosAccepted }}
+  email: {{ .Values.config.acme.emailAddress | quote }}
+{{- end }}
+
+workload_api:
+  socket_path: {{ include "spiffe-oidc-discovery-provider.workload-api-socket-path" . | quote }}
+  trust_domain: {{ include "spiffe-oidc-discovery-provider.trust-domain" . | quote }}
+
+health_checks:
+  bind_port: "8008"
+  ready_path: "/ready"
+  live_path: "/live"
+{{- end }}
```

Comment:
```suggestion
{{- end }}
{{- end }}
```

Reply:
Not following this. Why this additional end?

Reply:
Its the other end of the line 4 change:
```
{{- with .root }}
```